cms/src/components/content/ContentOverview.vue

```diff
@@ -77,6 +79,44 @@ if (!Object.entries(TagType).some((t) => t[1] == tagTypeString)) tagTypeString =
 
 const titleType = tagTypeString ? tagTypeString : props.docType;
 router.currentRoute.value.meta.title = `${capitaliseFirstLetter(titleType)} overview`;
+
+const searchTerm = ref("");
+
+const sortOptionsAsRef = ref(null);
```

Comment:
Please don't use `null`. Perhaps we should put that in an ADR 😅 But `null` in JavaScript is a bit problematic because `undefined` also exists, so if you want to check if something exists and only do `foo == undefined`, that might not return true if it's null. So instead, always initialize with `undefined` (if needed) and avoid the usage of `null`.

Reply:
Will keep that in mind, and change it to undefined. Thanks for the advice.

Reply:
I see this still needs to be changed to `undefined`.

Reply:
Changed